libs/application/ui-fields/src/lib/CompanySearchFormField/CompanySearchFormField.tsx

```diff
@@ -0,0 +1,135 @@
+import { gql, useLazyQuery } from '@apollo/client'
+import {
+  CompanySearchField,
+  FieldBaseProps,
+  formatText,
+} from '@island.is/application/core'
+import { Box } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import debounce from 'lodash/debounce'
+import React, { FC, useMemo, useState } from 'react'
+import { CompanySearchItem } from './CompanySearchItem'
+import kennitala from 'kennitala'
+import { CompanySearchController } from '@island.is/shared/form-fields'
+import { mockCompanies } from './__mock-data__/companies'
+
+export const COMPANY_REGISTRY_COMPANIES = gql`
+  query SearchCompanies($input: RskCompanyInfoSearchInput!) {
+    companyRegistryCompanies(input: $input) {
+      data {
+        name
+        nationalId
+      }
+    }
+  }
+`
+
+interface Props extends FieldBaseProps {
+  field: CompanySearchField
+}
+
+export const CompanySearchFormField: FC<Props> = ({ application, field }) => {
+  const {
+    id,
+    title,
+    placeholder,
+    setLabelToDataSchema = true,
+    useMockOptions = false,
```

Comment:
I would also like to get rid of useMockOptions and all mock code/data from our production bundles.

We need to find some other way to mock this data, or ask them for more test companies in our dev/staging environments if that helps.

Reply:
The reason why I did this was for storybook. We do get mock data from Skatturinn on dev but the component didn't work on storybook, might not have xroad configs? I'm not sure.

Reply:
Removed useMockOptions